src/__tests__/events.js

```diff
@@ -1,9 +1,10 @@
+import {eventMap, eventAliasMap} from '../event-map'
 import {fireEvent} from '..'
 
 const eventTypes = [
   {
     type: 'Clipboard',
-    events: ['copy', 'paste'],
+    events: ['copy', 'cut', 'paste'],
```

Comment:
Would be nice to derive more from `eventMap` to make sure missing events here aren't a concern, but this doesn't drive all event tests/isn't intended a complete reindexing of `eventMap`, so would require a bit of test rewriting that might be better at some other time to maintain momentary stability (plus when I have some actual time)